### PR TITLE
Wrap apps with the app name

### PR DESCRIPTION
**Pull request by sergiusens**
We wrap with the app name as the params are now globbed into the
wrapper

LP: #1534150

### CODE REVIEW

File: snapcraft/tests/test_meta.py [hunk 1]
```diff
@@ -209,6 +209,34 @@ def test_wrap_exe_must_write_wrapper(self, mock_assemble_env):
...
+        mock_assemble_env.return_value = """\
+PATH={0}/part1/install/usr/bin:{0}/part1/install/bin
+""".format(common.get_partsdir())
+
+        snapdir = common.get_snapdir()
+        os.mkdir(snapdir)
+
+        relative_exe_path = 'test_relexepath'
+        with open(os.path.join(snapdir, relative_exe_path), 'w'):
+            pass
```
Comment (kyrofa) [inline snapcraft/tests/test_meta.py]: Is this better than `open(os.path.join(snapdir, relative_exe_path), 'w').close()`?

**Comment by kyrofa** [comment]
Small ignorable comment-- looks good to me.

**Comment by elopio** [comment]
lgtm.
I want to get all merged to easily try all the binaries.

File: snapcraft/tests/test_meta.py (hunk 1 above)
Comment (sergiusens) [inline snapcraft/tests/test_meta.py]: no it's not, I'll fix this in this test and the test I copied from, thanks for spotting this!

**Pull request merged**